pydropsonde/circles.py

```diff
@@ -164,6 +164,11 @@ def apply_fit2d(self):
                 "Zonal gradient of " + name,
                 "Meridional gradient of " + name,
             ]
+            use_names = [
+                par + "_circle_mean",
+                "derivative_of_" + par + "_wrt_x",
```

Comment:
currently the variable is called 'dx' so it should be "derivative_of_" + par + "_wrt_dx", but I'd actually rename dx to x. as I kind of associate dx with a difference and not with a position; but the calculations later use it as a position.

Reply:
good idea, I switched dx and dy to be x and y